src/agent/onefuzz/src/syncdir.rs

```diff
@@ -203,7 +203,15 @@ impl SyncedDir {
                 let file_name = item
                     .file_name()
                     .ok_or_else(|| anyhow!("invalid file path"))?;
-                if ignore_dotfiles && file_name.to_string_lossy().starts_with('.') {
+                let file_name_str = file_name.to_string_lossy();
+
+                // explicitly ignore azcopy temporary files
+                // https://github.com/Azure/azure-storage-azcopy/blob/main/ste/xfer-remoteToLocal-file.go#L35
```

Comment:
👍

Reply:
I'm assuming there's currently no way to control the temporary path that `azcopy` uses.

Reply:
Yeah, looks like not: https://github.com/Azure/azure-storage-azcopy/blob/067b2fc3f79aca43a378ee1a6761268c7d1e4f38/ste/xfer-remoteToLocal-file.go#L471-L477

Reply:
It's for a good reason.  Generally speaking, rename of files in the same directory are atomic, where move is not.   It's significantly more complicated (and sometimes not possible) to ensure users get the benefit of downloading to a temp file then renaming if you have to use an alternate directory.